examples/tutorial/tut_allocator.cpp

```diff
@@ -9,11 +9,14 @@
 
 int main(int, char**)
 {
-  constexpr std::size_t SIZE = 1024;
-
+  // _umpire_tut_get_allocator_start
   auto& rm = umpire::ResourceManager::getInstance();
 
   umpire::Allocator allocator = rm.getAllocator("HOST");
+  // _umpire_tut_get_allocator_end
+
+  // _umpire_tut_de_allocate_start
```

Comment:
```suggestion

```

Reply:
I think I see what you are asking for.  Let me see if I can try something.

Reply:
I'm sorry, I haven't used this suggestion feature thingy before. Basically, I think moving the "// _umpire_tut_de_allocate_start" statement from line 18 to line 20 or so will make it only include the allocate + print statement + deallocate. (Kinda narrows the focus to only the allocate/deallocate which is what the description for the code snippet is talking about anyway).

Reply:
I've done this.  Thanks for your comments.  It will take about ten minutes for the document to be updated.  I appreciate your review and have narrowed the focus where you suggested.